src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
+/*
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PriseDay = 40;
+
+  if (days < 3) {
+    return days * PriseDay;
+  } else if (days >= 3 && days < 7) {
+    return days * PriseDay - 20;
```

Comment:
Remove the `else` keyword as it is unnecessary when the preceding `if` block contains a `return` statement.